src/basic/NESTED_INIT.cpp

```diff
@@ -117,8 +139,7 @@ void NESTED_INIT::runKernel(VariantID vid)
       startTimer();
       for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-#if 0
-// using collapse here doesn't appear to yield a performance benefit
+#if defined(USE_OMP_COLLAPSE)
```

Comment:
Should we note that USE_OMP_COLLAPSE is #undef'ed further up? This could cause some confusion.

Reply:
It's mainly for experimentation.  I follow this pattern in several other places in the code.